hadoop-hdds/docs/content/concept/Containers.zh.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Container
+weight: 5
+menu: 
+  main:
+     parent: 概念
+summary: 容器的相关描述，它是Ozone的基本复制单元.
+---
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+容器是 Ozone/HDDS 的基本复制单元，由 Storage Container Manager (SCM) 服务管理。
+
+容器是可以包含多个块（block）的大型二进制单元：
+
+![Containers](Containers.png)
+
+块作为本地信息，不由 SCM 管理。因此，即使在系统中创建了数十亿个小文件（即创建了数十亿个块），Datanode也仅报告容器的状态，以及复制容器。
+
+当 Ozone Manager 向 SCM 请求分配一个新的块时，SCM 将识别一个合适的容器，并且产生一个包含 `ContainerId` + `LocalId` 的块Id，客户端则连接到存储这个容器的 Datanode 上进行读写操作。而 Datanode 则能够基于 `LocalId` 管理这个独立的块。
```

Comment:
```suggestion
当 Ozone Manager 向 SCM 请求分配一个新的块时，SCM 将识别一个合适的容器，并且产生一个包含 `容器 ID` + `本地 ID` 的块 ID。基于 `本地 ID`，能使客户端连接到存储这个容器的数据节点上，数据节点也能够管理这个独立的块。
```

Reply:
Considering that they are the variable name in the project code,  the translation of the `ContainerId` and  the`LocalId` should be maintained here?

Reply:
This suggestion is based on [this page](https://ci-hadoop.apache.org/view/Hadoop%20Ozone/job/ozone-doc-master/lastSuccessfulBuild/artifact/hadoop-hdds/docs/public/zh/concept/datanodes.html).
I think it just means that `Block ID` consists of `Container ID` and `Local ID`.

Reply:
I mainly consider that the backtick indicates that it is a code reference.  It seems more appropriate to stay the same.